setup/utils/build_protos.py

```
@@ -120,12 +107,10 @@ def make_proto(path):
 
 def main():
```

Is main() still needed if it can't be run directly? Maybe return an error that says run from setup.py?

---

I opted to remove it for simplicity sake.